notebooks/codesearchnet-opennmt.py

```diff
@@ -1,36 +1,39 @@
+"""
+CLI tool for converting CodeSearchNet dataset to OpenNMT format for
+function name suggestion task.
+
+Usage example:
+    wget 'https://s3.amazonaws.com/code-search-net/CodeSearchNet/v2/java.zip'
+    unzip java.zip
+    python notebooks/codesearchnet-opennmt.py \
+        --data_dir='java/final/jsonl/valid' \
```

Comment:
```suggestion
        --data-dir='java/final/jsonl/valid'
```
We probably need to change the arg below also!